CMakeLists.txt

```diff
@@ -52,9 +52,11 @@ include( ${ITK_USE_FILE} )
 #==================================================================
 # Look for OpenCV (required by some plugins, but not the main app)
 #==================================================================
-set( AutoOpenCVPath ${IBIS_EXTERNAL_DEPENDENCIES_DIR}/opencv-${IBIS_OPENCV_LONG_VERSION}/install/share/OpenCV )
-find_package( OpenCV REQUIRED PATHS ${AutoOpenCVPath} )
-include_directories( ${OpenCV_INCLUDE_DIRS} )
+if( DEFINED OpenCV_DIR )
```

Comment:
Not sure I understand this. find_package actually looks for OpenCV and sets OpenCV_DIR if it is not already done, so it is kind of weird to look for opencv only if OpenCV_DIR is defined. What happens if you don't do a super build. You need to define OpenCV_DIR manually for Ibis to use OpenCV? It would be much simpler to have an option (IBIS_USE_OPENCV). If the option is ON, then you look for OpenCV. In cases where OpenCV is installed in the standard path of the system, path will be found automatically. In the case where IBIS_EXTERNAL_DEPENDENCIES_DIR is correctly set, path will also be found automatically.